### ISSUE TITLE
Air date vs Chronological order.

### ISSUE THREAD

**Issue by Poofey** [AceFire6/ordered-arrowverse#403]
Flash s2e18 "All star teamup" aired after Arrow s3e18 "public enemy" however, chronologically it happens between arrow s3e16 and e17. Even though the sites sorts episodes with air dates would it be feasible to sort them so that these events happen in order? it's just weird to see felicity and team arrow getting in trouble with the law and atom almost dying only to immediately see them in a crossover episode like everythings fine.  And I'm sure it's not the only time that events get jumbled with air dates.

**Comment by ChippewaBarr** [comment]
May as well tag on here.

Flash S01E22 is chronologically incorrect. It takes place after Arrow S03E20 but before Arrow S03E21, and also after Flash S01E20 but before S01E22.

In the episode Eddie is still captured (until they find him in the episode) and Oliver is still "Oliver" and hasn't fully "assimilated" into Ras al Ghul's heir but still wears the League of Assassin's garb while helping Barry - and also gives him a heads up that he may need Barry's help soon.

**Comment by Christian-Stieber** [comment]
Yeah, and I just noticed that Supergirl's 1x18 "Worlds Finest" is listed before Barry is doing the Tachyon device experiment... which happens in Flash 2x18. The Supergirl episode kind of happens in the middle of the Flash episode, so the absolutely perfectly perfect way to watch is to pause Flash mid-episode and continue later, but I guess watching Supergirl after Flash would be fine too :-)
